integration/runner/runner.go

```diff
@@ -204,7 +204,7 @@ func Run() error {
 
 	// Build cAdvisor.
 	glog.Infof("Building cAdvisor...")
-	err := RunCommand("godep", "go", "build", "github.com/google/cadvisor")
+	err := RunCommand("godep", "go", "build", "--ldflags='-extldflags \"-static\"'", "github.com/google/cadvisor")
```

Comment:
@vishh Do you have any clue what to write here to make it work?
I tried a lot of different combinations, but the linker just didn't parse it.